nbgrader/alembic/versions/167914646830_added_task_cells.py

```diff
@@ -0,0 +1,116 @@
+"""added task cells
+
+Revision ID: 167914646830
+Revises: 724cde206c17
+Create Date: 2018-06-23 07:46:30.221922
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '167914646830'
+down_revision = '724cde206c17'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
Ah, I see. I think in that case you might not have to use alembic at all if the table is automatically created. But we might want to check this with a different flavor of sql database like Postgres just to be sure that's not sqlite behavior.

Reply:
(I should say, the point of alembic is for database migrations, as you've inferred, so it can automatically modify an old database to be compatible with a new spec. Usually what happens if, say, you add a new column, is that when nbgrader would try to access the database, it would throw an error, hence the need for alembic. But in the context of nbgrader the migrations should always happen through `nbgrader upgrade` rather directly through alembic itself.)